src/metabase/public_settings.clj

```diff
@@ -85,7 +85,7 @@
   ;; Don't i18n this docstring because it's not user-facing! :)
   "Unique identifier used for this instance of Metabase. This is set once and only once the first time it is fetched via
   its magic getter. Nice!"
-  :visibility :internal
+  :visibility :authenticated
   :setter     :none
```

Comment:
`site-uuid` setting is read-only, so it shouldn't be possible to change it.
Example: sending `PUT /api/setting/side-uuid` will result in `500` response saying that `site-uuid` is readonly